### ISSUE TITLE
[Kotlin] 1.26.0 NPE on VoidRequest success flow

### ISSUE THREAD

**Issue by AngelaQiao** [auth0/Auth0.Android#341]
### Description

NPE occurs during run time when using VoidRequest and response received is successful -https://github.com/auth0/Auth0.Android/blob/master/auth0/src/main/java/com/auth0/android/request/internal/VoidRequest.java#L57

Happens as this is annotated non-null:
https://github.com/auth0/Auth0.Android/blob/master/auth0/src/main/java/com/auth0/android/callback/BaseCallback.java#L41

Exception:
```
uncaughtException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter payload
    java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter payload
```

### Reproduction
Set up env config and use a valid email with `passwordlessWithEmail` method to get a success response from the `passwordless/start` API
```kotlin
val client: AuthenticationAPIClient ...
client.passwordlessWithEmail("email@email.com", PasswordlessType.ANDROID_LINK)
      .start(object :  AuthenticationCallback<Void> { 
     override fun onSuccess(payload: Void) { 
       ...
     }
      ...
      } )
```

### Environment

- **Version of this library used:** 1.26.0
- **Version of the platform or framework used, if applicable:** Android
- **Other relevant versions (language, server software, OS, browser):**
- **Other modules/plugins/libraries that might be involved:**

**Comment by lbalmaceda** [comment]
Thanks for bringing this up @AngelaQiao. 
I tried reproducing it with Java and it seems to work OK.

```java
getAuthAPIClient()
     .passwordlessWithEmail("asd@asd.asd", PasswordlessType.CODE)
     .start(new AuthenticationCallback<Void>() {
            @Override
            public void onSuccess(@NonNull Void payload) {
                Log.e("TAG", "SUCCESS");
            }

            @Override
            public void onFailure(@NonNull AuthenticationException error) {
                Log.e("TAG", "FAILED!", error.getCause());
            }
        });
```

output `com.auth0.androidsample E: SUCCESS`

Which confirms why the unit tests didn't catch this. 

When `1.26.0` is used on a Kotlin app, it fails as you described. Seems like the non-null annotated parameters are also being checked in runtime internally. I'll roll back the changes in the annotations for the uses of `Void` in the callbacks.

**Comment by lbalmaceda** [comment]
Should be fixed on `1.26.1`.

**Comment by AngelaQiao** [comment]
Thanks @lbalmaceda 😄 
It's now not crashing for me
